src/views/about/about.jsx

```diff
@@ -102,8 +133,26 @@ var About = React.createClass({
 
                         <li>
                             <h3><FormattedMessage id='about.support' /></h3>
-                            <p><FormattedHTMLMessage id='about.supportDescription' /></p>
-                        </li>
+                            <p><FormattedMessage id='about.supportDescription' /></p>
+                            values={{
+        supportersList: 'National Science Foundation, Scratch Foundation, Siegel Family Endowment, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, MacArthur Foundation',
```

Comment:
Shorten this line somehow. A quick search didn't turn up any obvious ways to break up a line in JSON though - any ideas @chrisgarrity?

Reply:
Yeah.

Reply:
you can add `// eslint-disable-next-line max-len` just before the long line to disable the check.

Reply:
Okay 👍